airbyte-integrations/connectors/source-sendgrid/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_sendgrid"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "==0.51.39"
+airbyte-cdk = "==0.65.0"
```

Comment:
How do we feel about setting this to `<1.0`?

Reply:
There is a version in the manifest. There looks to be an explicit check they are the same. That was only when I was behind though. Maybe the CDK can be more ahead? cc @girarda